vsphere/resource_vsphere_host.go

```diff
@@ -817,3 +913,206 @@ func (h HostAccessManager) ChangeLockdownMode(ctx context.Context, mode types.Ho
 	_, err := methods.ChangeLockdownMode(ctx, h.Client(), &req)
 	return err
 }
+
+func resourceVSphereHostUpdateServices(d *schema.ResourceData, meta interface{}, oldVal, newVal interface{}) error {
+	client := meta.(*Client).vimClient
+	hostID := d.Id()
+	hostObject, err := hostsystem.FromID(client, hostID)
+	if err != nil {
+		return fmt.Errorf("error while retrieving HostSystem object for host ID %s. Error: %s", hostID, err)
+	}
+
+	servicesSet, ok := d.Get("services").(*schema.Set)
+	if !ok {
+		return fmt.Errorf("error reading 'services': type assertion to *schema.Set failed")
+	}
+
+	updatedServices := make([]interface{}, 0) // Prepare to collect updated services configurations
+
+	services := servicesSet.List()
+	for _, service := range services {
+		serviceMap := service.(map[string]interface{})
+		updatedServiceMap := make(map[string]interface{}) // Prepare to collect updated service configuration
+		return fmt.Errorf("error retrieving HostSystem object for host ID %s. Error: %s", hostID, err)
```

Comment:
this return seems out of place, maybe a copy-paste gone wrong

Reply:
removed the return

Reply:
i actually moved it down to be above the call for it